src/mgxs_header.F90

```diff
@@ -496,6 +498,18 @@ subroutine mgxsiso_init_file(this, node_xsdata, groups, get_kfiss, get_fiss, &
                        (scatt_coeffs(imu - 1, gout, gin) + &
                         scatt_coeffs(imu, gout, gin))
                 end if
+                ! Now create CDF from fmu with the analytical integral of a
+                ! piecewise linear function
+                if (imu > 1) then
+                  p0 = scatt_coeffs(imu - 1, gout, gin)
+                  mu0 = mu - dmu
+                  m = (scatt_coeffs(imu, gout, gin) - &
+                       scatt_coeffs(imu - 1, gout, gin)) / dmu
+                  norm = norm + &
+                       HALF * m * mu * mu + &
+                       (p0 - m * mu0) * mu + &
+                       (HALF * m * mu0 * mu0 - p0 * mu0)
+                end if
```

Comment:
Yes, totally does. This was a mistake.  And a bug-inducing mistake (since norm wasn't zeroed).  Same as your later comment on scattdata_header.F90 too.

There was a period where I was toying with integrating the legendre's exactly for the CDF, and then making the PDF match as needed.  I then decided that was probably too much effort for too little reward.  This was leftover from my final reel-back.